README.md

```diff
@@ -145,6 +146,20 @@ Attach to member property declarations of a struct, actor, or class that `@Membe
 * `@Init(.public, label: String)`
   <br> Custom labels can be combined with all other behaviors.
 
+* `@Init(assignee: String)`
+  <br> Override the target property to which the initializer argument should be assigned. By default, a property named “property” has the assignee `self.property`, as demonstrated in `self.property = property`.
+
+* `@Init(type: Any.Type)`
+  <br> Override the type of the argument in the initializer.
+
+* `@Init(assignee: String, type: Any.Type)`
+  <br> Combine `assignee` and `type` to support usage with property wrappers:
+
+  ```swift
+  @Init(assignee: "self._isOn", type: Binding<Bool>)
```

Comment:
Can the macro not read the property name and assume it's always `self._propertyName`? Are there caveats to this? 🤔

Reply:
Oh I see that `type` is more of a general purpose thing than anything specifically related to property wrappers. Interesting.

I wonder if we can have a specific property wrapper argument somehow. Like:

```swift
@Init(propertyWrapperType: Binding<Bool>)
```

Reply:
Usually, I think that would work. But, MemberwiseInit would technically have to assume that `@Binding` is indeed a `SwiftUI.@Binding` property wrapper and not some other attribute/macro. I've considered adding a `mode` that could make that assumption reasonable:

```swift
import SwiftUI

@MemberwiseInit(.public, mode: .swiftUI)
struct CounterView: View {
  @Binding public var count = 0

  var body: some View { … }
}
```

I haven't completely thought through the implications there. My primary concern is how to support multiple versions of Swift and SwiftUI over time.

Thanks, and definitely do let me know your thoughts!

Reply:
(Our latest comments crossed.) That's an interesting idea! I'll think some on that tonight. I definitely agree that providing the assignee as a string is the most awkward part of the current solution.

Reply:
I started #14 with some draft thoughts on how to better support property-wrapped members.